packages/next-app/src/app/metadata/[network]/pool/[poolId]/(components)/MetadataAttributesTable/index.tsx

```diff
@@ -123,14 +123,29 @@ function Row({
           </div>
         )}
       </Td>
-      <Td className="min-w-[10rem]">{data.key}</Td>
+      <Td className="min-w-[5rem] max-w-[10rem]">
+        {!data.value || !data.description ? (
+          <div className="flex justify-between">
+            <>{data.key}</>
+            <span className="inline-flex items-center rounded-md bg-red-200 p-0.5 text-xs font-semibold text-red-600">
+              NEEDS EDIT
+            </span>
```

Comment:
This looks like a badge, should we use that instead? Also, wdyt about using the radix color here instead of TW's?

Reply:
I don't know much about the Radix colors but I definitely can to try using it